File: test_models.py
import urllib.request, json
req = urllib.request.Request('https://openrouter.ai/api/v1/models')
with urllib.request.urlopen(req) as response:
    data = json.loads(response.read().decode())
    for model in data['data']:
        if 'image' in model['id'].lower() or 'flux' in model['id'].lower() or 'imagen' in model['id'].lower() or 'diffusion' in model['id'].lower():
            print(model['id'])

File: test_models.js
fetch("https://openrouter.ai/api/v1/models")
  .then(r => r.json())
  .then(d => {
     let c = 0;
     for(let m of d.data) {
       if(m.id.includes("flux") || m.id.includes("imagen") || m.id.includes("image")) {
           console.log(m.id);
           c++;
       }
     }
     console.log("Total matched:", c);
  });
